qa/L0_sequence_batcher/sequence_batcher_test.py

```diff
@@ -80,7 +80,7 @@
     res = []
```

Comment:
Earlier in this file there is another case where custom is used as the only ragged batch testcase. @GuanLuo this seems like another case of missed ragged batching testing? We should file a ticket to track these and get them enabled.

Reply:
Agree. We have L0_batch_input which tests ragged batching as well. We may want to either extract the related tests to one test folder so we can manage it in one place or place the names of backend with ragged support in a common area (test util?) and the tests will read the backends to be tested from that common area.